### PR TITLE
Update setup docs

### LINKED ISSUE #254: Document installation from conda-forge
The package is now on conda, but this isn't documented in the readme or on the website.

### PR DESCRIPTION
**Pull request by lochhh**
## Description

**What is this PR**

- [ ] Bug fix
- [ ] Addition of a new feature
- [x] Other

**Why is this PR needed?**
This PR closes #254 and #262

**What does this PR do?**
This PR:
- updates the documentation to include conda installation instructions
- replaces pip install instructions in repo README quick install with conda instructions
- updates any mention of python 3.10 in the setup to python 3.11

## References
#254 #262

## How has this PR been tested?
Installed movement from conda locally and ran `movement info`
Docs built locally and on CI

## Checklist:

- [x] The code has been tested locally
- [ ] Tests have been added to cover all new functionality
- [x] The documentation has been updated to reflect any changes
- [x] The code has been formatted with [pre-commit](https://pre-commit.com/)

### CODE REVIEW

File: docs/source/getting_started/installation.md [hunk 1]
```diff
@@ -1,68 +1,76 @@
...
+::::::{tab-set}
+:::::{tab-item} Users
+::::{tab-set}
+:::{tab-item} Conda
+Create and activate an environment with movement installed.
+```sh
+conda create -n movement-env -c conda-forge movement
+conda activate movement-env
+```
+(Optional) Update an existing installation of movement to the latest version within the same environment:
```
Comment (sfmig) [inline docs/source/getting_started/installation.md]: We could move the update part outside of both tabs, to reuse the common sentence. Right now it also feels a bit hidden.

I had a go [here](https://github.com/neuroinformatics-unit/movement/pull/264/commits/b96ee86ca028a6484d97df7c186213457efb674a) but again feel free to revert if you disagree. Below how it looks:

image

**Review by sfmig — APPROVED** [review]
thanks @lochhh for updating! yay for spotting the inconsistencies 🦅 

Mostly minor comments on phrasing, which you can ofc take or leave. I had a go at using nested tabs, to retain the previous users/developers split, and also separated install and upgrade instructions. I like it, but feel free to edit or revert, it was mostly to see how it looked.

Re `movement` vs movement: I will dump my current thoughts in PR #266. For this PR I think it makes sense to just make it consistent with itself. If we later decide to go monospace we will need to search / review everything anyways.

**Comment by lochhh** [comment]
Thanks @sfmig for the suggestions! I agree that the update instructions seem hidden and love the idea of splitting install and update! 

Regarding the users/developers tabs, I'm thinking now, if we should just redirect developers to the contributing guide, as
- the contributing guide has the same but more complete instructions (with the additional step to initialise the precommit hooks), 
- this increases maintainability (we don't need to update the duplicate info in two places)

If we go with the "redirect developers" method, should we have these in separate subheadings, rather than having a Developers tab which they click into, only to find that they need to click again (this is just my personal preference though 🤔). I also think it looks cleaner like this: 
image

Lmk your thoughts! 😄

**Comment by sfmig** [comment]
TLDR: I agree with everything 😁 

> I'm thinking now, if we should just redirect developers to the contributing guide

Yes, good call! Much better for the reasons you say

> If we go with the "redirect developers" method, should we have these in separate subheadings

Yes, it looks great! Feels more spacious ✨ 👩‍🚀  (and as you say less clicking, yay!)

> I also changed the update instructions to recommend starting fresh instead, to prevent potential incompatibility issues - happy to revert if this is unnecessary.

Again good call, love the clarity!

Thanks @lochhh, it's looking neat! 🚀
